AUTHORS

```diff
@@ -53,17 +53,22 @@ Camilo Schoeningh                       camilo.schoeningh@dunnhumby.com
 Canh Ngo                                canhnt@gmail.com
 Carlos Castro                           carlos.castro@jumo.world
 Changyu Wang                            changyuwang@tencent.com
+Charles-Edouard Brétéché                charled.breteche@gmail.com
 Charles-Henri Guérin                    charles-henri.guerin@zenika.com
 chenyahui                               chenyahui9@jd.com
+chenyaqi01                              chenyaqi01@baidu.com
+chez-shanpu                             tomoki.sugiura@mail.shanpu.info
```

Comment:
Tomoki Sugiura is already lower down in the AUTHORS, I think we just need to bump mailmap to associate this address also?